docs/book/misc/debugging.md

```diff
@@ -0,0 +1,229 @@
+---
+description: A guide to debug common issues and get help.
+---
+
+# Debugging Guide
+This page documents a series of how-tos for efficient debugging and getting help without losing your hair.
+
+## 💡 Introduction
+If you stumbled upon this page, chances are you're facing issues with using ZenML. First, thank you for your interest in ZenML!
+We want to make your journey with ZenML as easy and smooth as possible when faced with issues.
+This page documents suggestions and best practices to let you debug, get help, and solve issues quickly.
+
+## 🔍 Getting help
+The most direct and quickest path to get help is on our Slack. 
+Join our Slack channel [here](https://zenml.io/slack-invte).
```

Comment:
Typo . /slack-invite. Btw from now on we also have /slack as another link to the same

Reply:
Fixed!